initramfs/init.sh

```diff
@@ -139,8 +136,13 @@ critical_error()
 
 boot_root()
 {
-    echo "Mounting ${root}."
-    mount -t "${rootfstype}" -o exec,suid,dev,noatime,"${rootflags},${rwmode}" "${root}" "${ROOT_MOUNT}"
+    root_device=${root}
```

Comment:
Should add quotes root_device="${root}", Shellcheck will complain about this, this prevents globbing.